src/style.css

```
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1:last-child,
+.stars--2:last-child,
+.stars--3:last-child,
+.stars--4:last-child,
+.stars--5:last-child {
```

you can use `.stars`
```suggestion
.stars :last-child {
```